atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClient.java

```diff
@@ -99,19 +106,22 @@ CASResult cas(TableReference tableReference,
             ConsistencyLevel commit_consistency_level)
             throws InvalidRequestException, UnavailableException, TimedOutException, org.apache.thrift.TException;
 
+    @Timed
     public CASResult put_unless_exists(TableReference tableReference,
             ByteBuffer key,
             List<Column> updates,
             ConsistencyLevel serial_consistency_level,
             ConsistencyLevel commit_consistency_level)
             throws InvalidRequestException, UnavailableException, TimedOutException, org.apache.thrift.TException;
 
+    @Timed
```

Comment:
I _think_ this can be removed (see earlier comment)

Reply:
Removed. If we need it we can bring it back